### ISSUE TITLE
No amperage limit function

### ISSUE THREAD

**Issue by tihoangyeudau** [awawa-dev/HyperSerialEsp8266#15]
Can you add amperage limiter to avoid overpowering of the power supply?

Thank you!

**Comment by awawa-dev** [comment]
No, I don't plan such function because we can't properly implement measurement for such feature. And particularly I don't want to hear complains about situations, where "safe" amperage value was set in the software, but the power supply (e.g. which was already at the limit but the ambient temperature rose a bit so the effectiveness gone down) broke down along few other connected in-line components like Raspberry Pi, the grabber and finally the TV or the amplifier in the worst case scenario 😉
